backend/Rides/clusters_qh.py

```diff
@@ -0,0 +1,41 @@
+from Rides.DataAccess.rides_db_context import Session
+from Rides.Domain.clusters import Clusters
+
+session = Session()
+
+
+def get_clusters(limit: int = None):
+    clusters = session.query(Clusters.Id, Clusters.MinLat,
+                             Clusters.MinLng, Clusters.MaxLat, Clusters.MaxLng)
+    if limit is not None:
+        clusters = clusters.limit(limit=limit)
+    else:
+        clusters = clusters.all()
+
+    res = []
+    for c in clusters:
+        cluster = {
+            'Id': c[0],
+            'MinLat': c[1],
+            'MinLng': c[2],
+            'MaxLat': c[3],
+            'MaxLng': c[4],
+        }
+        res.append(cluster)
+    return res
+
+
+def get_clusters_center(limit: int = None):
+    clusters = get_clusters(limit)
+
+    res = []
+    for c in clusters:
+        centerLat = (c['MaxLat'] - c['MinLat'])/2 + c['MinLat']
```

Comment:
I'm considering holding centers as a separate field in DB, I will add it in next PR.